data/Application.css

```
@@ -50,3 +50,13 @@ vte-terminal {
     background-color: #fff;
     color: #000;
 }
+
+.color-custom radio {
+    background-color: #fff;
+    color: #000;
+    -gtk-icon-source: -gtk-icontheme("list-add");
```

We should make sure to use the -symbolic variant here
```suggestion
    -gtk-icon-source: -gtk-icontheme("list-add-symbolic");
```

---

Could you elaborate on when to use which variant? Is there some documentation on this somewhere?

---

Basically the `-symbolic` variant will follow the text color whereas the normal variants are full color icons. It's only kind of a historical coincidence that the normal variant for `list-add` happens to be close to the text color in the light style. This will change in a future revision of the icon set. So in this context where we want a simple shape icon that follows the UI text color, `symbolic` is the way to go